dashboard/config/levels/custom/dance/hoai-pilot-6-first-dance.level

```diff
@@ -72,9 +94,7 @@
     ],
     "encrypted_examples": "QcyE1JnIa2c7fdVSHVOkbatQQV4OdsWG9/WbEyZMlHVc9qqXh0SPTAxAdr7o\nYoaj2AvdWdpsweQIUzGmgIErkA==\n",
     "uses_lab2": "true",
-    "generate_dancer_mode": "false",
-    "ai_code_generate": "false",
-    "ai_code_preview": "true",
+    "guide_mode": "instructions",
```

Comment:
I think we'll need to update the [Dance AI level editor fields](https://github.com/code-dot-org/code-dot-org/blob/03f12fa625fc4e9618c2641914476ede2d484186/dashboard/app/views/levels/editors/fields/_dance_ai_fields.html.haml) and the [dance level file](https://github.com/code-dot-org/code-dot-org/blob/03f12fa625fc4e9618c2641914476ede2d484186/dashboard/app/models/levels/dancelab.rb) with the new keys, right?

Reply:
Great catches.
- Updated the keys.
- Updated the editor UI.  Just shows a text field for `guide_mode` for now but could be updated to a dropdown later.

Also updated all three of the final level's sublevels.